iOS/DuckDuckGo/AppServices/DBPService.swift

```diff
@@ -66,7 +66,7 @@ final class DBPService: NSObject {
     }
 }
 
-private final class DBPFeatureFlagger: RemoteBrokerDeliveryFeatureFlagging {
+final class DBPFeatureFlagger: RemoteBrokerDeliveryFeatureFlagging {
```

Comment:
This is the one public interface change in this PR – I feel mostly okay about it since the feature flagger class is just an interface to the dependency provider and thus I find it acceptable if various parts of the code are creating their own instance, but I would be happy to have the initialization happen elsewhere and keep this private.